[Docs] Add the docs for code string generation (#653)
mckinsey/vizro: vizro-ai/docs/pages/user-guides/retrieve-dashboard-code.md

# Retrieve the code of Vizro-AI generated dashboard

!!! warning "Feature in Alpha Version"
    This feature is currently in **Alpha** version. It is stable but not officially released yet. We welcome feedback via the [GitHub Issues](https://github.com/mckinsey/vizro/issues). Note that the feature and its API may still undergo minor adjustments.

> This guide shows how to retrieve the code of Vizro-AI generated dashboard, for further iterations and deployment.

Reviewer: ```suggestion
This guide shows how to retrieve the code of a Vizro-AI generated dashboard. The code can be used for further iterations, improvements and deployment.
```

> While Vizro-AI can follow complex user requirements well and generate high-quality dashboards, due to the nature of LLMs, the generated dashboards often approximately match user expectations but may not be exact. Besides refining the user prompt and rerunning Vizro-AI, you can also extract the code and iterate manually to achieve the desired result.

Reviewer: ```suggestion
While Vizro-AI can follow complex user requirements well and generate high-quality dashboards, due to the nature of LLMs, the generated dashboards often only approximately match user expectations. Besides refining the user prompt and re-running Vizro-AI, you can also extract the code and iterate manually to achieve the desired result.
```
